docs/src/pages/[platform]/components/pagination/react.mdx

```diff
@@ -84,15 +84,19 @@ It's common to use a paged API where the total number of pages in the dataset is
 
 ## Accessibility
 
-By default, the root node of the Pagination component has a role of `navigation`. The previous-page button `<` has `aria-label="Go to previous page"` by default, and likewise, the next-page button `>` has `aria-label=Go to next page`. The current page is labelled by an invisible text [`Current Page:`](https://www.w3.org/WAI/tutorials/menus/structure/#indicate-the-current-item) in conjunction with the current page number (e.g, `Current Page: 1` if page `1` is the current page). Other pages by default have `aria-label=Go to page X`, where `X` is the number of the page. To customize these labels:
+Accessible labels are provided for the pagination buttons either via the `aria-label` attribute or by using the [VisuallyHidden](/react/components/visuallyhidden) primitive.
 
-- `previousLabel`: Set the `aria-label` for the left arrow button `<` (defaults to `Go to previous page`)
+Use the following props to customize these labels:
 
-- `nextLabel`: Set the `aria-label` for the right arrow button `>` (defaults to `Go to next page`)
+- **`previousLabel`**: Set the `aria-label` for the previous page button (defaults to `Go to previous page`)
 
-- `currentPageLabel`: Set the invisible label for current page (defaults to `Current Page:`). It will be used to construct the label text for current page. e.g, `Current Page: 1` if page `1` is the current page.
+- **`nextLabel`**: Set the `aria-label` for the next page button (defaults to `Go to next page`)
 
-- `pageLabel`: Set the label for each page button other than the current page (defaults to `Go to page`). It will be used to construct the `aria-label`. e.g, `Go to page 1` for page `1` button.
+- **`currentPageLabel`**: Set the `VisuallyHidden` label for current page (defaults to `Current Page:`). This will be used to construct the label text for current page. e.g, `Current Page: 1` if page `1` is the current page.
```

Comment:
defaults to `Current Page:` or `Current Page`?

Reply:
[`Current Page:`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/primitives/shared/constants.ts#L527) is used as our default